### PR TITLE
Register new package Augmentor.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [Evizero/Augmentor.jl](https://github.com/Evizero/Augmentor.jl)
Release: [v0.0.1](https://github.com/Evizero/Augmentor.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/Evizero/Augmentor.jl/builds/233755089)
cc: @Evizero

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@Evizero This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by Evizero** [comment]
@tkelman Just FYI I use the same color-dependent test code here as you proposed for https://github.com/JuliaImages/ImageTransformations.jl/pull/32 and this time the JuliaCIBot was run and succeeded. Thanks again for the suggestion.

File: Augmentor/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+CoordinateTransformations 0.4.0
+Interpolations 0.6
+Rotations
+StaticArrays
+OffsetArrays
+IdentityRanges
+ColorTypes 0.4
+ShowItLikeYouBuildIt
+FileIO
+Compat 0.19
```
Comment (tkelman) [inline Augmentor/versions/0.0.1/requires]: not using IndexStyle, only need 0.17 for abstract type

Comment (Evizero) [inline Augmentor/versions/0.0.1/requires]: I'll fix that real quick and overwrite the tag

**Comment by Evizero** [comment]
hold on with the merge. I am getting the most confusion travis error suddenly. All tests pass, but then it says `Your build exited with 1.` and the travis job is marked as failed

https://travis-ci.org/Evizero/Augmentor.jl/jobs/234477342

**Comment by Evizero** [comment]
Ok false alarm, so I restarted a test on an old commit that previously succeeded and it now fails for the same reason. So this issue, while still weird and troubling, seems to be unrelated to Augmentor.

**Comment by tkelman** [comment]
try setting the travis git clone depth really high https://docs.travis-ci.com/user/customizing-the-build#Git-Clone-Depth

**Comment by Evizero** [comment]
That seems to have solved it. Do you know what is happening?

**Comment by tkelman** [comment]
it's a bug in git or travis

**Comment by tkelman** [comment]
Is the code here derived from the Python version at all? That looks like it's also MIT licensed, but always better to acknowledge in the license file if anything is derived.

**Comment by Evizero** [comment]
No derived work in either direction. He is a friend of mine and we developed two disjoint versions in parallel

**Pull request merged**